README.md

```diff
@@ -168,6 +169,20 @@ with the command `pca-wizard-templates  --emails`.
 | interface_type | Type of interface GoPhish will use with mail server. | string | "SMTP" | no |
 | ignore_cert | Indicate if GoPhish should ignore certs with mail server. | boolean | `True` | no |
 
+## User Report Field Dictionary ##
+
+User report JSONs are also exported by `gophish-export`. User reports
+summarize data from targeted user clicks on phishing emails generated by a
+campaign. These JSON files are later imported into the PCA database.
```

Comment:
Remove reference to PCA- this repo should stand on it's own as a Gophish helper utility, regardless of whether or not it is used for PCAs.
```suggestion
campaign.
```

Reply:
See 2d96175613954e3c341aa38614d80369113217af